client/src/app/+videos/+video-watch/video-watch.component.scss

```diff
@@ -58,6 +58,16 @@ $video-info-margin-left: 44px;
     display: flex;
     justify-content: center;
     flex-grow: 1;
+
+    ::ng-deep .video-js {
```

Comment:
Put this in https://github.com/Chocobozzz/PeerTube/blob/develop/client/src/sass/player/peertube-skin.scss no?

Reply:
I made the changes you asked, but I don't understand these tests errors :-/